core/consensus/grandpa/impl/voting_round_impl.cpp

```diff
@@ -911,7 +919,8 @@ namespace kagome::consensus::grandpa {
       if (result == outcome::failure(VotingRoundError::DUPLICATED_VOTE)) {
         return false;
       }
-      env_->onCompleted(result.as_failure());
+      // TODO(xDimon): Check if it is really needed
```

Comment:
Another approach to deal with commented code is to file an issue.
For example, at the beginning of the ages, the Iroha team had the rule to supply any todo with a task id. I much appreciate that approach, because orphaned todos without an issue id are going to be hung unsolved forever.